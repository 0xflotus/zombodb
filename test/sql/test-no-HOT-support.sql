CREATE TABLE no_hot AS SELECT * FROM events WHERE id = 1;
UPDATE no_hot SET id = id WHERE id = 1;
CREATE INDEX idxno_hot ON no_hot USING zombodb ((no_hot.*));
DROP TABLE no_hot CASCADE;