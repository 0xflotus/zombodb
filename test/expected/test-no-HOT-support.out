CREATE TABLE no_hot AS SELECT * FROM events WHERE id = 1;
UPDATE no_hot SET id = id WHERE id = 1;
CREATE INDEX idxno_hot ON no_hot USING zombodb ((no_hot.*));
ERROR:  Heap Only Tuple (HOT) found at (0, 1).  Run VACUUM FULL no_hot; and then create the index
DROP TABLE no_hot CASCADE;
